src/main/java/codeu/model/data/User.java

```diff
@@ -24,44 +24,86 @@
   private final String passwordHash;
   private final Instant creation;
   private final Boolean isAdmin;
+  private final String bio;
+  private final String instagramHandle;
+  private final String snapchatHandle;
 
   /**
    * Constructs a new User.
    *
-   * @param id the ID of this User
-   * @param name the username of this User
-   * @param passwordHash the password hash of this User
-   * @param creation the creation time of this User
+   * @param id              the ID of this User
+   * @param name            the username of this User
+   * @param passwordHash    the password hash of this User
+   * @param creation        the creation time of this User
    */
   public User(UUID id, String name, String passwordHash, Instant creation, Boolean isAdmin) {
-    this.id = id;
-    this.name = name;
-    this.passwordHash = passwordHash;
-    this.creation = creation;
-    this.isAdmin = isAdmin;
+    this(id, name, passwordHash, creation, isAdmin, null, null, null);
   }
 
-  /** Returns the ID of this User. */
+    /**
+     * Constructs a new User.
+     *
+     * @param id              the ID of this User
+     * @param name            the username of this User
+     * @param passwordHash    the password hash of this User
+     * @param creation        the creation time of this User
+     * @param bio             the bio of the user
+     * @param instagramHandle the instagram username of the user
+     * @param snapchatHandle  the snapchat username of the user
+     */
+    public User(UUID id, String name, String passwordHash, Instant creation, Boolean isAdmin, String bio, String instagramHandle, String snapchatHandle) {
```

Comment:
@blessinga98 This is the new constructor. Notice that this constructor is what finally sets the instance variables of the class